engine/execution/computation/computer/computer.go

```diff
@@ -377,3 +397,23 @@ func (bc *blockCommitter) Run() {
 func (bc *blockCommitter) Commit(view state.View) {
 	bc.views <- view
 }
+
+type evensHasher struct {
```

Comment:
```suggestion
type eventHasher struct {
```
shouldn't it be `eventHasher`?

Reply:
Of course! Good catch!